renku-theme-dev/renku-realm.json

```diff
@@ -1325,7 +1325,7 @@
       "name": "Terms and Conditions",
       "providerId": "terms_and_conditions",
```

Comment:
nitpick: this is not uppercase, like the other actions.

Reply:
This file is created by exporting the realm from Keycloak -- I did not set these values. 

<img width="673" alt="image" src="https://github.com/SwissDataScienceCenter/keycloak-theme/assets/1196411/09833183-c609-44bc-9e07-7320e8d75099">

The attribute was changed to uppercase in version 21.0.0, but this was considered an accident and was reverted back to to lowercase in a bug fix version.

https://www.keycloak.org/docs/23.0.6/upgrading/#migrating-to-21-0-2